towncrier.toml

```diff
@@ -0,0 +1,28 @@
+[tool.towncrier]
+directory = "changelog.d"
+filename = "CHANGELOG.md"
+start_string = "<!-- towncrier release notes start -->\n"
+underlines = ["", "", ""]
+title_format = "## [v{version}] - {project_date}"
+issue_format = "[#{issue}](https://github.com/Uninett/Argus-frontend/issues/{issue})"
+wrap = true
+
+[[tool.towncrier.type]]
+directory = "critical"
+name = "Critical"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "added"
+name = "Added"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "changed"
+name = "Changed"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "fixed"
+name = "Fixed"
+showcontent = true
```

Comment:
Really, the types defined by "Keep a changelog" are the ones we usually use:

- `Added` for new features.
- `Changed` for changes in existing functionality.
- `Deprecated` for soon-to-be removed features.
- `Removed `for now removed features.
- `Fixed` for any bug fixes.
- `Security` in case of vulnerabilities.